ui/src/components/executions/Resume.vue

```diff
@@ -97,6 +100,16 @@
                         this.$toast().success(this.$t("resumed done"));
                     });
             },
+            cancel() {
+                this.$store
+                    .dispatch("execution/kill", {
+                        id: this.execution.id,
+                    })
+                    .then(() => {
+                        this.isDrawerOpen = false;
+                        this.$toast().success(this.$t("cancel done"));
+                    });
+            },
```

Comment:
Added a cancel method for the cancel button to use. This runs execution kill and displays a "cancel done" message.